### DISCUSSION TITLE
Simple image viewer as default

### DISCUSSION THREAD

**Discussion by AndyP2002** [WinMerge/winmerge#2423] [Q&A]
Newbie question.... 
I am trying to use WinMerge to compare two folders of camera images and videos. I would like to just double click on  a file name and view the image or video using a simple image or video viewer (preferably one built into Windows, such as Windows Photos or Photo Gallery, and Media Player or similar for videos). 
I'm looking for behaviour like Windows File Explorer where the application used is determined by the file extension of the file clicked on. 
Is this possible?

I have found that in 'Options' I can change the 'External Editor' to (for example) my graphics editing app (Corel PhotoPaint), but that obviously only works for image files, and it also annoyingly opens a new copy of the app for every image.

Am I missing something simple? I do realise that I can right-click on a file and select "open left" or "open right" / "with...." "and select a program from a list, but it is very time consuming to do that for every file.

**Comment by sdottaka** [comment]
Unfortunately, this isn't possible with WinMerge at the moment.

FreeFileSync seems to be able to open files in fewer steps than WinMerge.

Also, although it's probably not what you want, WinMerge has built-in image and webpage comparison features,
so you can use the image comparison feature to display images, and the webpage comparison feature to display videos.

To display videos in the webpage comparison feature by double-clicking them, specify the file extension as a regular expression, as follows:

![image](https://github.com/user-attachments/assets/342ae7fe-7154-4a2e-9add-58dc2d66a6e0)

![image](https://github.com/user-attachments/assets/8d1a0f87-4537-4648-9a0b-530f20656f8e)
